runtime/src/read_only_accounts_cache.rs

```diff
@@ -44,9 +45,14 @@ impl ReadOnlyAccountsCache {
             hits: AtomicU64::new(0),
             misses: AtomicU64::new(0),
             lru: Arc::new(RwLock::new(Vec::new())),
+            per_account_size: Self::per_account_size(),
         }
     }
 
+    fn per_account_size() -> usize {
+        std::mem::size_of::<ReadOnlyAccountCacheEntry>() + std::mem::size_of::<RwLock<Instant>>()
```

Comment:
Could we add a comment about why we need to add and extra `RwLock<Instant>` size to `ReadOnlyAccountCacheEntry>`? Looks like `ReadOnlyAccountCacheEntry` already has a `last_used: RwLock<Instant>` field?